src/Repository/Criteria.php

```diff
@@ -159,19 +157,14 @@ public function setDomain(string $name): self
 
     public function addLanguage(string $lang): self
     {
-        if (null === $this->languages) {
-            $this->languages = [];
-        }
-        array_push($this->languages, $lang);
+        $this->languages[] = $lang;
```

Comment:
When you attempt to access an array element that doesn't exist, PHP automatically initializes the array for you.